lib/web/apiserver_test.go

```diff
@@ -1920,13 +1920,13 @@ func TestWebAgentForward(t *testing.T) {
 }
 
 func TestActiveSessions(t *testing.T) {
+	// Use enterprise license (required for moderated sessions).
+	modules.SetTestModules(t, &modules.TestModules{TestBuildType: modules.BuildEnterprise})
```

Comment:
We can't do this since other tests that do not require special features will receive the wrong module content. I.e. Kubernetes now enforces a valid license when upserting `kube_servers`
This affects  https://github.com/gravitational/teleport/pull/24118

If the test changes the modules it cannot run in // with other tests

Reply:
ups looks like you already fixed it

Reply:
It's no longer a parallel test, so we should be good to go.